packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js

```diff
@@ -312,3 +317,15 @@ function getCellContents(sheet, columnIndex, rowIndex) {
   const cell = sheet[xlsx.utils.encode_cell({ c: columnIndex, r: rowIndex })];
   return cell === undefined ? '' : cell.v; // Extract the value of the cell if it wasn't blank
 }
+
+async function getSurveyFromSheet(models, sheet) {
+  const firstSurveyResponseId = getColumnHeader(sheet, INFO_COLUMN_HEADERS.length);
+  if (!firstSurveyResponseId) {
+    throw new Error(
+      'Each tab of the import file must have at least one previously submitted survey as the first entry',
+    );
+  }
+
+  const firstSurveyResponse = await models.surveyResponse.findById(firstSurveyResponseId);
+  return models.survey.findById(firstSurveyResponse.survey_id);
+}
```

Comment:
This was removed when the new Admin Panel feature was merged into the epic branch, not sure why but it is still needed when we're just updating existing survey responses

Reply:
Sorry @chris-bes this was probably my poor conflict resolution!